client/app/mfaEnrollment/mfaEnrollment.html

```diff
@@ -0,0 +1,34 @@
+<div class="mfa-enrollment-page container-fluid h-100 position-absolute overflow-auto">
+    <div class="row minh-100 justify-content-md-center">
+        <div class="col-md-7 p-4 p-md-5 bg-white minh-100">
+            <div class="p-0 p-md-5">
+                <div class="text-center mb-3">
+                    <img
+                        src="images/double-authantification.png"
+                        srcset="images/double-authantification@2x.png x2, images/double-authantification@3x.png x3"
+                        class="img-fluid"
+                        alt="" />
+                </div>
+
+                <h1 class="oui-heading_4" data-translate="mfa_enrollment_title"></h1>
+                <p data-translate="mfa_enrollment_info1"></p>
+                <p data-translate="mfa_enrollment_info2"></p>
+
+                <div class="mx-auto my-5 float-right">
+                    <oui-button class="m-2" on-click="$ctrl.goBack()">
```

Comment:
```suggestion
                    <oui-button class="m-2" data-on-click="$ctrl.goBack()">
```

Reply:
Let's discuss about it! Not mandatory.

Reply:
We've already talked about it at least 50 times but why not do it another time 😅